From 29fac5e7535407115b7246f6d92020f61b9964cd Mon Sep 17 00:00:00 2001
From: Haohui Mai <wheat9@apache.org>
Date: Wed, 4 Nov 2015 09:16:43 -0800
Subject: [PATCH 1074/2748] HDFS-9357. NN UI renders icons of decommissioned
 DN incorrectly. Contributed by Surendra Singh
 Lilhore.

(cherry picked from commit b61aa716e8c2dc26f889f846b6bb84b90aecd5af)

Change-Id: I8560f131a7cd7cddddb828a80685df318878f9d3
---
 .../src/main/webapps/hdfs/dfshealth.html           |    4 ++--
 .../hadoop-hdfs/src/main/webapps/static/hadoop.css |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/hdfs/dfshealth.html b/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/hdfs/dfshealth.html
index 90a52d9..8937c00 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/hdfs/dfshealth.html
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/hdfs/dfshealth.html
@@ -286,8 +286,8 @@
   <ul class="dfshealth-node-legend">
     <li class="dfshealth-node-icon dfshealth-node-alive">In service</li>
     <li class="dfshealth-node-icon dfshealth-node-down">Down</li>
-    <li class="dfshealth-node-icon dfshealth-node-decommisioned">Decommisioned</li>
-    <li class="dfshealth-node-icon dfshealth-node-down-decommisioned">Decommissioned &amp; dead</li>
+    <li class="dfshealth-node-icon dfshealth-node-decommissioned">Decommissioned</li>
+    <li class="dfshealth-node-icon dfshealth-node-down-decommissioned">Decommissioned &amp; dead</li>
   </ul>
 </div>
 <div class="page-header"><h1><small>In operation</small></h1></div>
diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/hadoop.css b/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/hadoop.css
index a5ebef0..a596f6d 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/hadoop.css
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/hadoop.css
@@ -235,7 +235,7 @@ header.bs-docs-nav, header.bs-docs-nav .navbar-brand {
     content: "\e013";
 }
 
-.dfshealth-node-decommisioned:before {
+.dfshealth-node-decommissioned:before {
     color: #eea236;
     content: "\e136";
 }
@@ -245,7 +245,7 @@ header.bs-docs-nav, header.bs-docs-nav .navbar-brand {
     content: "\e101";
 }
 
-.dfshealth-node-down-decommisioned:before {
+.dfshealth-node-down-decommissioned:before {
     color: #2e6da6;
     content: "\e017";
 }
-- 
1.7.9.5

